.changeset/good-foxes-build.md

```diff
@@ -0,0 +1,5 @@
+---
+"robot3": major
+---
+
+Release version 1.0.0
```

Comment:
Can you change this to be a description of this change? Maybe something like a code example would help.

Reply:
@matthewp Done. Better like this?